server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/CassandraBlobModule.java

```diff
@@ -65,5 +66,12 @@
             .addPartitionKey(BucketBlobParts.ID, DataType.text())
             .addClusteringColumn(BucketBlobTable.NUMBER_OF_CHUNK, DataType.cint()))
 
+        .table(DumbBlobCache.TABLE_NAME)
+        .comment("Holds information for cache all blob parts composing" +
+            "Messages` headers and bodies are stored as blobparts.")
```

Comment:
"Write through cache for small blobs stored in the object storage"

Reply:
"stored in a slower blob store implementation"